examples/eks_cluster_webshop/eks.tf

```diff
@@ -0,0 +1,100 @@
+#2. create EKS cluster
+module "eks" {
+  source  = "terraform-aws-modules/eks/aws"
+  version = "18.20.5"
+
+  cluster_name    = var.cluster_name
+  cluster_version = "1.21"
```

Comment:
1.21 is quite old already. 1.22 maybe? Not sure if this can break anything.
[Kubernetes version 1.22 prerequisites](https://docs.aws.amazon.com/eks/latest/userguide/update-cluster.html#update-1.22)